### ISSUE TITLE
jQuery.uniqueSort example does not work in Chrome.

### ISSUE THREAD

**Issue by KevSlashNull** [jquery/api.jquery.com#1062]
While researching about the uniqueSort method, I noticed that that method does not work on the [example page](https://api.jquery.com/jQuery.uniqueSort/) in Chrome.

![image](https://user-images.githubusercontent.com/28510368/30773573-9a5e9682-a073-11e7-94bd-293bfb7a9997.png)

![image](https://user-images.githubusercontent.com/28510368/30773576-a9783d1c-a073-11e7-9851-a3e1179a6e2c.png)

KevSlashNull. \^-^

**Comment by dmethvin** [comment]
Yep, the example should include jQuery version 3.0 or higher. Thanks!

**Comment by AurelioDeRosa** [comment]
Actually, we could even specify version 1.12 which is where it was introduced. The root cause of the problem is that [our template](https://github.com/jquery/api.jquery.com/blob/master/entries2html.xsl) always assume version 1.10.2.

@dmethvin how do you feel about introducing a new template for those methods that will only work in future versions of jQuery?

**Comment by mgol** [comment]
@AurelioDeRosa Ideally every example except for methods removed in latest version would use the latest jQuery version (it shouldn't get auto-updated, though). If people look at the example and see it working and then try it in their project and see it fail it's not encouraging.

Such a move might render some examples broken, though. We'd then have to depend on community members to report any such issues.
